### PR TITLE
support delete range kv command

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

File: proto/errorpb.proto [hunk 1]
```diff
@@ -29,6 +29,14 @@ message KeyNotInRegion {
     optional bytes end_key     = 4;
 }
 
+message RangeNotInRegion {
```
Comment (disksing) [inline proto/errorpb.proto]: How about return _KeyNotInRegion_ if _start_key_ or _end_key_ is not in region?

Comment (zhangjinpeng1987) [inline proto/errorpb.proto]: They are different.

**Comment by zhangjinpeng1987** [comment]
@zimulala PTAL

**Comment by zhangjinpeng1987** [comment]
@disksing PTAL

**Comment by zimulala** [comment]
LGTM

**Comment by disksing** [comment]
LGTM.

**Pull request merged**